### PR TITLE
Added Chinese (Traditional) to Thank You Section

### LINKED ISSUE #320: Add Chinese (Traditional) to the Thank You Section
**Actual Behaviour**

The current Thank You section contains only Chinese (Simplified). However, people in Hong Kong/Taiwan use primarily Chinese (Traditional), which should be added.

**Expected Behaviour**

A message in Chinese (Traditional) should be added.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Ryan10145**
- [X] Included a Preview link and screenshot showing after and before the changes.
- [X] Included a description of the change below.
- [X] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://ryan10145.github.io/gci18.fossasia.org/)

- Fixes #320 

## Description / Changes

- Adds a message in Chinese (Traditional) for those who primarily use it over Chinese (Simplified)
- Increased delay on Thank You Section to make it easier to read more complex scripts like Chinese

## Screenshots if any:

screen shot 2018-11-12 at 1 50 31 am

- - - - - - - - - - - -

### CODE REVIEW

File: js/thankyou.js [hunk 1]
```diff
@@ -36,7 +37,7 @@ var typingEffect = function typingEffect() {
    setTimeout(function () {
      app.innerHTML = '';
      typingEffect();
-   }, 1000);
```
Comment (sk9331657) [inline js/thankyou.js]: @Ryan10145  Why did you change the timeout??

Comment (Ryan10145) [reply js/thankyou.js]: @sk9331657 Specifically when I was trying to read the Chinese, the language would change when I was halfway through. So I changed the timeout since I felt that for more complicated character based languages, there wasn't enough time given to read.

Comment (sk9331657) [reply js/thankyou.js]: Ok @Ryan10145
